CSS varialbes and small tweaks

AkselNLadegaard opened the pull request.

Commit:
New preferences

Switched to CSS variables, as they cascade.
Fixed a few warnings.

Commit:
Exact versioning

Commit:
Fixed and updated packages.

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**persocit-main**

:tada: Your build was successful! [See the Deploy preview here.](https://persocitmain-cssvariables.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/2b382612-b6c0-4ae4-87aa-15f5104db39d/sites/84eac81a-cd0b-44fe-b42f-7915414fcf2c/builds/68a5a31c-1bac-4ddc-92ee-820c3b80a94e/details)

:clock1: Build time: **1m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :green_heart: 100   |
| Accessibility |   :large_orange_diamond: 87   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 73   |

:link: [View full report](https://build-68a5a31c-1bac-4ddc-92ee-820c3b80a94e.gtsb.io/reports/lighthouse/index.html)

Commit:
Dark mode 

Which listens for you browser to change preference, and stores to local storage, and is pretty nifty.

Commit:
Now checks whether or not it is in a browser 

Required for gatsby build to work.

Commit:
Now also excluding document references

Document is unavailable during the node.js build & tests

Commit:
Entire darkmode isn't rendered if not in a browser
